src/cmd/shared.go

```diff
@@ -35,13 +36,15 @@ func getAppConfig() (types.AppConfig, error) {
 	return appConfig, nil
 }
 
-func getAwsConfig() (types.AwsConfig, error) {
+func getAwsConfig(homeDir, profile string) (types.AwsConfig, error) {
```

Comment:
Can we get the home dir from within this function instead of requiring it to be passed in?